public/components/checkbox.js

```diff
@@ -19,19 +19,22 @@ export const checkbox = (props, children) => h('div', {
       'sr-only': true,
     },
   }),
-  h('span', {
-    class: {
-      'fa-stack': true,
-    },
-  }, [
-    h('i', { class: 'far fa-circle fa-stack-2x' }),
-    props.checked && h('i', { class: 'fas fa-check fa-stack-1x text-green-500' }),
-  ]),
   children && h('label', {
     for: props.id,
     class: {
       'flex-grow': true,
       'leading-tight': true,
+      'flex': true,
+      'flex-row': true,
+      'items-center': true,
     },
-  }, children),
+  }, [
+    h('span', {
+      class: {
+        'fa-stack': true,
+      },
+    }, [
+      h('i', { class: 'far fa-circle fa-stack-2x' }),
+      props.checked && h('i', { class: 'fas fa-check fa-stack-1x text-green-500' }),
+    ]), ...children]),
```

Comment:
Everything looks good, and if you're alright with me nit-picking just a little, can you change this last part to:

```
  ]),
  ...children,
])
```

I found it just a smidge hard to follow with `...children` on the same line as a sibling.

Reply:
Sure, will push in a couple minutes